lib/workers/repository/process/lookup/index.ts

```diff
@@ -201,11 +203,21 @@ export async function lookupUpdates(
         buckets[bucket] = [release];
       }
     }
+    const depResultConfig = mergeChildConfig(config, res);
     for (const [bucket, releases] of Object.entries(buckets)) {
       const sortedReleases = releases.sort((r1, r2) =>
         versioning.sortVersions(r1.version, r2.version)
       );
-      const release = sortedReleases.pop();
+      const { release, pendingChecks, pendingReleases } = filterInternalChecks(
+        depResultConfig,
+        versioning,
+        bucket,
+        sortedReleases
+      );
+      // istanbul ignore next
+      if (!release) {
```

Comment:
Can this happen? Then we should better test it.

Reply:
I couldn't work out a way to reproduce it in tests, but I want to keep it here just in case.